examples/nlp/language_modeling/conf/megatron_gpt_config.yaml

```diff
@@ -30,6 +30,7 @@ exp_manager:
     name: null
   resume_if_exists: True
   resume_ignore_no_checkpoint: True
+  resume_from_checkpoint: ${model.resume_from_checkpoint} 
```

Comment:
@maanug-nv do we still want to keep model.resume_from_checkpoint ?

Reply:
I kept it for now so that it won't break any user's code that uses model.resume_from_checkpoint. I think there will be another effort very soon to reduce the config complexity across all NeMo LLM, so we can leave this item for that.